pypsa/pf.py

```diff
@@ -1187,7 +1190,9 @@ def calculate_B_H(sub_network, skip_pre=False):
             for c in sub_network.iterate_components(network.passive_branch_components)
         ]
     )
-    sub_network.p_branch_shift = np.multiply(-b, phase_shift, where=b != np.inf)
+    sub_network.p_branch_shift = (
+        -b * phase_shift
+    )  # np.multiply(-b, phase_shift, where=b != np.inf)
```

Comment:
Could you briefly explain why `where=b != np.inf` is no longer needed?

And remove old code.